dataland-e2etests/src/test/kotlin/org/dataland/e2etests/MetaDataControllerTest.kt

```diff
@@ -0,0 +1,81 @@
+package org.dataland.e2etests
+
+import org.dataland.datalandbackend.openApiClient.api.CompanyDataControllerApi
+import org.dataland.datalandbackend.openApiClient.api.EuTaxonomyDataControllerApi
+import org.dataland.datalandbackend.openApiClient.api.MetaDataControllerApi
+import org.dataland.datalandbackend.openApiClient.model.CompanyAssociatedDataEuTaxonomyData
+import org.dataland.datalandbackend.openApiClient.model.DataMetaInformation
+import org.dataland.datalandbackend.openApiClient.model.PostCompanyRequestBody
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class MetaDataControllerTest {
+
+    private val basePathToDatalandProxy = "http://proxy:80/api"
```

Comment:
kann man das `private final val` deklarieren denke ich mal

Reply:
Ich habe vermutet es gibt Unterschied zwischen val und final beim Komplieren. Aber man benutzt anscheinened nur val und und var in Kotlin.
passt so :)